public/app/shared/services/formFormatters/formatBirthday.dir.js

```diff
@@ -1,27 +1,98 @@
-// @deprecated
-// (function() {
+'use strict';
+ angular
+   .module('iserveu')
+   .controller('birthdayController',['$scope', function ($scope) {
 
-//  angular
-//    .module('iserveu')
-//    .directive('formatBirthday', ['$filter', formatBirthday]);
-
-//  function formatBirthday($filter) {
-
-//    return {
-//        require: 'ngModel',
-//        link: function(scope, element, attrs, ngModelController) {
-
-//        // ngModelController.$parsers.push(function(data) {
-//        //  return new Date(data);
-//        // });
-
-//            ngModelController.$formatters.push(function(data) {
-//          var d = new Date(data);
-//          return new Date(d.setTime( d.getTime() +
-//          d.getTimezoneOffset()*60000 ));
-//          });
-//        }
-//    };
-//  }
-
-// })();
+     	$scope.days = [];
+        $scope.months =[{
+            value: 1,
+            name: 'January'
+        }, {
+            value: 2,
+            name: 'February'
+        }, {
+            value: 3,
+            name: 'March'
+        }, {
+            value: 4,
+            name: 'April'
+        }, {
+            value: 5,
+            name: 'May'
+        }, {
+            value: 6,
+            name: 'June'
+        }, {
+            value: 7,
+            name: 'July'
+        }, {
+            value: 8,
+            name: 'August'
+        }, {
+            value: 9,
+            name: 'September'
+        }, {
+            value: 10,
+            name: 'October'
+        }, {
+            value: 11,
+            name: 'November'
+        }, {
+            value: 12,
+            name: 'December'
+        }];
+;
+        $scope.years = createYearsArray();
+        
+        function createYearsArray(){
+        	//maximum 120 years.
+        	var today = new Date();
+        	var currentYear = today.getFullYear();
+        	var years=[];
+        	while(years.push(currentYear--)<=120){};
+        	return years;
+        }
+        //
+        //make it beautiful here by creating code to initialize days.
+        $scope.selectYear = function(month,year){
```

Comment:
http://stackoverflow.com/questions/1184334/get-number-days-in-a-specified-month-using-javascript